tests/unit/api/main.py

```diff
@@ -1185,29 +1188,42 @@ class Test_switch_register:
     def test_basic(self):
         """Calling switch_register should create an object in the db."""
         api.switch_register('sw0', type=MOCK_SWITCH_TYPE,
-                username="switch_user", password="switch_pass", hostname="switchname")
+                            username="switch_user",
+                            password="switch_pass",
+                            hostname="switchname")
         assert model.Switch.query.one().label == 'sw0'
 
     def test_duplicate(self):
-        """switch_register should complain if asked to make a duplicate switch."""
+        """
+        switch_register should complain if asked to make a duplicate switch.
+        """
```

Comment:
This actually introduces a pep257 violation; can we avoid that? Leaving the comment on the same line as the opening `"""`, and ony wrapping the closing one might be okay.